### DISCUSSION TITLE
my log, i need help

### DISCUSSION THREAD

**Discussion by jr614** [dnGrep/dnGrep#999] [Q&A] [answered]
[Grep_Error_Log.zip](https://github.com/dnGrep/dnGrep/files/13462605/Grep_Error_Log.zip)

hello everyone, can you tell me how to fix

**Comment by doug24** [comment, marked as answer]
You need Administrator access to modify files in Program Files.  It may work if you start dnGrep 'Run as administrator', or if that does not work, copy the files you want to modify to another folder, do the changes there, then manually copy the files back to the original folder (so you can do the copy as administrator.

**Reply by jr614** [reply]
thank you :)
